python/GafferOFX/__init__.py

```
@@ -0,0 +1,50 @@
+
+import os
+import pathlib
+
+__import__( "Gaffer" )
+__import__( "GafferImage" )
+
+if hasattr( os, "add_dll_directory" ) :
+	os.add_dll_directory( ( pathlib.Path( os.environ["OFX_ROOT"] ) / "lib" ).resolve() )
+del os, pathlib # Don't pollute the namespace
+
+from ._GafferOFX import *
```

Ruff (0.8.2)

`from ._GafferOFX import *` used; unable to detect undefined names

(F403)